service/internal/auth/casbin.go

```diff
@@ -89,6 +89,8 @@ p,	role:admin,		/v1/token/authorization,						  				        *,      allow
 p,	role:standard,		policy.*,																read,			allow
 p,	role:standard,		kasregistry.*,													read,			allow
 p,	role:standard,    kas.AccessService/Rewrap, 			           *,			allow
+p,      role:standard,          kas.AuthorizationService/GetDecisions,             *,                   allow
+p,      role:standard,          kas.AuthorizationService/GetDecisionsByToken,      *,                   allow
```

Comment:
```suggestion
p,      role:standard,          kas.AuthorizationService/GetDecisions,             read,                   allow
p,      role:standard,          kas.AuthorizationService/GetDecisionsByToken,      read,                   allow
```

Reply:
@mkleene just aligning it to the `Get` prefix to be scoped in case there is ever introduction of `SetDecision` we don't accidentally expose it.